xrpl/models/ledger_objects/ripple_state.py

```diff
@@ -19,21 +19,87 @@ class RippleState(LedgerObject):
+    balance indicates that the high account holds tokens issued by the low account. The
+    issuer in this is always set to the neutral value ACCOUNT_ONE. This field is
+    required.
+    """
+
     high_limit: IssuedCurrencyAmount = REQUIRED  # type: ignore
+    """
+    The limit that the high account has set on the trust line. The `issuer` is the
+    address of the high account that set this limit. This field is required.
+    """
+
+    low_limit: IssuedCurrencyAmount = REQUIRED  # type: ignore
+    """
+    The limit that the low account has set on the trust line. The `issuer` is the
+    address of the low account that set this limit. This field is required.
+    """
+
     previous_txn_id: str = REQUIRED  # type: ignore
+    """
+    The identifying hash of the transaction that most recently modified this entry.
+    This field is required.
+    """
+
     previous_txn_lgr_seq: int = REQUIRED  # type: ignore
+    """
+    The index of the ledger that contains the transaction that most recently modified
+    this entry. This field is required.
+    """
+
     high_node: Optional[str] = None
-    low_node: Optional[str] = None
+    """
+    (Omitted in some historical ledgers) A hint indicating which page of the high
+    account's owner directory links to this entry, in case the directory consists of
+    multiple pages.
+    """
+
     high_quality_in: Optional[int] = None
+    """
+    The inbound quality set by the high account, as an integer in the implied ratio
+    `HighQualityIn`:1,000,000,000. As a special case, the value 0 is equivalent to 1
+    billion, or face value.
+    """
+
     high_quality_out: Optional[int] = None
+    """
+    The outbound quality set by the high account, as an integer in the implied ratio
+    `HighQualityOut`:1,000,000,000. As a special case, the value 0 is equivalent to 1
+    billion, or face value.
+    """
+
+    low_node: Optional[str] = None
+    """
+    (Omitted in some historical ledgers) A hint indicating which page of the low
+    account's owner directory links to this entry, in case the directory consists of
+    multiple pages.
+    """
+
     low_quality_in: Optional[int] = None
+    """
+    The inbound quality set by the low account, as an integer in the implied ratio
+    `LowQualityIn`:1,000,000,000. As a special case, the value 0 is equivalent to 1
+    billion, or face value.
+    """
+
     low_quality_out: Optional[int] = None
+    """
+    The outbound quality set by the low account, as an integer in the implied ratio
+    `LowQualityOut`:1,000,000,000. As a special case, the value 0 is equivalent to 1
+    billion, or face value.
+    """
+
     ledger_entry_type: LedgerEntryType = field(
         default=LedgerEntryType.RIPPLE_STATE,
         init=False,
     )
+    """
+    The value `0x0072`, mapped to the string `RippleState`, indicates that this is a
+    RippleState entry.
+    """
 
 
 class RippleStateFlag(Enum):
```

Comment:
This class is not used in other parts of the code. The same is true for `SignerListFlag` class.

Reply:
It doesn't need to be.